src/qtiItem/core/Loader.js

```diff
@@ -221,10 +252,14 @@ var Loader = Class.extend({
             }
         });
     },
-    buildResponse(data) {
+    buildResponse(data, responseRule) {
         const response = this.buildElement(data);
 
-        response.template = data.howMatch || null;
+        response.template = responseHelper.getTemplateUriFromName(
+            responseHelper.getTemplateNameFromResponseRules(data.identifier, responseRule)
```

Comment:
As I see in this part 
![image](https://user-images.githubusercontent.com/25976342/92480050-0ca67480-f1ed-11ea-9d83-b036f2007b2a.png)
a template can be inside `this.item.responses[responseKey].template` in case `responseRules.length > 0`
Is this case is handled?

Reply:
here item responses initialized. before call to this method there is no this.item.responses yet